### PR TITLE
chore: update latest AFG revision with X25519 WK

### PR DESCRIPTION
**Pull request by Baha-sk**
Signed-off-by: Baha Shaaban <baha.shaaban@securekey.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -96,3 +96,10 @@ clean-build:
 	@rm -Rf ./.build
 	@rm -Rf ./test/bdd/docker-compose.log
 
+.PHONY: clean-test-keys
```
Comment (DRK3) [inline Makefile:99]: Should this be part of `make all`?

Comment (Baha-sk) [reply Makefile:99]: it can, I just added to support refreshing keys, I can add it to `make all`

Comment (Baha-sk) [reply Makefile:99]: added

**Review by DRK3 — APPROVED** [review]

**Comment by DRK3** [comment]
The title of the PR has a typo in it

**Comment by Baha-sk** [comment]
> The title of the PR has a typo in it

thanks @DRK3, updated :-)

**Review by sandrask — APPROVED** [review]

**Review by rolsonquadras — APPROVED** [review]

**Pull request merged**